js/script.js

```diff
@@ -7,7 +7,7 @@ const BUTTON_UP_COLOR = "#a0a0a0";
 const BUTTON_UP_OUTLINE = "";
 const BUTTON_DOWN_COLOR = "#f0f0f0";
 const BUTTON_DOWN_OUTLINE = "1px solid blue";
-const CANVAS_INIT_COLOR = palette_color_array[0];
+const CANVAS_INIT_COLOR = "transparent";  // Or "rgba(0,0,0,0)"
```

Comment:
Awesome! 👍 You can probably remove `rgba("0,0,0,0")` since we don't need

Reply:
ok, upđate in [8706a7e](https://github.com/Kully/pixel-paint/pull/23/commits/8706a7ebe9887f4e60b1b2e71e3bcc1c3c03349a)